### ISSUE TITLE
[Bug]: "Restart and Install" always available but does nothing

### ISSUE THREAD

**Issue by harryfear** [getferdi/ferdi#2258]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/getferdi/ferdi/blob/develop/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/getferdi/ferdi/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/getferdi/ferdi/issues) for a feature request that matches the one I want to file, without success.

### Ferdi Version

5.6.3

### What operating system are you using?

macOS

### Server

Ferdi

### Operating System Version

12.0.1 (21A559)

### What arch are you using?

arm64 (including Apple Silicon)

### Last Known working version of Ferdi

A few months ago

### Steps to reproduce, Expected Behavior, Actual Behavior

• Go to Settings > Updates
• Find "Restart & Install update" button is always available (when it is not expected to appear because there is no new version available)

OR

• Go to Settings > Updates
• Clicking the "Restart & Install update" button does nothing other than close Ferdi (when it is expected to do an upgrade).

In conclusion: Ferdi should update itself without issue, but the only way to update is to manually download the latest version.

![Screenshot 2021-11-19 at 16 39 17](https://user-images.githubusercontent.com/907604/142649747-9ef2ba81-e960-4d6d-ba32-a15f38d73e3c.png)

![Screenshot 2021-11-19 at 16 50 36](https://user-images.githubusercontent.com/907604/142651640-c0b4d199-f25c-4549-987a-a2da6a06daff.png)

**Comment by vraravam** [comment]
Please search for older duplicates before logging new tickets.

This is a duplicate of #329

**Comment by harryfear** [comment]
I did search but actually #329 didn't appear with my search query 😊

**Comment by vraravam** [comment]
please search for closed tickets as well - lots of duplicates on this specific issue
